packages/web-react/src/components/ScrollView/__tests__/useScrollViewStyleProps.test.ts

```diff
@@ -0,0 +1,24 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useScrollViewStyleProps } from '../useScrollViewStyleProps';
+
+describe('useScrollViewStyleProps', () => {
+  it('should return defaults', () => {
+    const { result } = renderHook(() =>
+      useScrollViewStyleProps({
+        direction: 'horizontal',
+        indicators: 'shadows',
+        isScrollbarDisabled: true,
+        isScrolledAtEnd: true,
+        isScrolledAtStart: true,
+      }),
+    );
+
+    expect(result.current.classProps).toBeDefined();
+    expect(result.current.classProps.root).toBe(
+      'ScrollView ScrollView--horizontal ScrollView--scrollbarDisabled is-scrolled-at-end is-scrolled-at-start',
+    );
+    expect(result.current.classProps.viewport).toBe('ScrollView__viewport');
+    expect(result.current.classProps.content).toBe('ScrollView__content');
+    expect(result.current.classProps.indicators).toBe('ScrollView__indicators ScrollView__indicators--shadows');
```

Comment:
I would love to see test to test `both` too, but 🤷

Reply:
done